views/guide.lavender

```diff
@@ -297,10 +297,13 @@ block content
 
                 if FALSE
                   div this isn't rendered
+
                 elseif FALSE
```

Comment:
jade syntax shows a space between 'else' and 'if', have you tried that?  maybe our lavender documentation is wrong.  @sugarPhlox can you confirm?